site-src/v1alpha2/references/policy-attachment.md

```diff
@@ -0,0 +1,294 @@
+# Policy Attachment
```

Comment:
As a reader, I think it would help me if Policy is defined early on in this document.
I can't seem to comprehend what we mean by policy. But then as soon as I read the GKEServicePolicy example, it starts to make sense.

Reply:
Good idea, added a paragraph to start this section off, let me know what you think.